packages/ga4/README.md

```diff
@@ -137,3 +137,8 @@ You must enable `autoTrack` to use `ga4` in this way, see [Onload](#onload) for
 # Acknowledgement
 
 This package builds on the great work done by [David Künnen](https://github.com/DavidKuennen) and [Dariusz Więckiewicz](https://github.com/idarek). David's work on the small drop in replacement for Universal Analytics, and Dariusz's port over to GA4 were instrumental in providing a blue print for `@minimal-analytics/ga4`.
+
+# Integration
```

Comment:
I feel like this would probably be better placed in an `INTEGRATIONS.md` file, that way we can scale the examples without bloating the core `README` file.

How about we create the new file, and link to it instead from this point in the `README`?